lib/datadog/tracing/contrib/ext.rb

```diff
@@ -18,6 +18,10 @@ module RPC
           TAG_SYSTEM = 'rpc.system'
           TAG_SERVICE = 'rpc.service'
           TAG_METHOD = 'rpc.method'
+
+          TAG_GRPC_STATUS_CODE = 'rpc.grpc.status_code'
+          TAG_GRPC_PACKAGE = 'rpc.grpc.package'
+          TAG_GRPC_FULL_METHOD = 'rpc.grpc.full_method'
```

Comment:
`rpc.grpc.full_method` isn't an agreed upon tag. I think you mean rpc.grpc.path which is the entire method path

Reply:
as discussed offline, this tag will be named `rpc.grpc.full_method`